Sources/View/NibOwnerLoadable.swift

```diff
@@ -41,16 +41,17 @@ public extension NibOwnerLoadable where Self: UIView {
                       Defaults to a brand new instance if not provided.
    - returns: A `NibOwnLoadable`, `UIView` instance
    */
-  static func loadFromNib(owner owner: Self = Self()) -> Self {
-    let layoutAttributes: [NSLayoutAttribute] = [.Top, .Leading, .Bottom, .Trailing]
-    for view in nib.instantiateWithOwner(owner, options: nil) {
+  @discardableResult
+  static func loadFromNib(owner: Self = Self()) -> Self {
+    let layoutAttributes: [NSLayoutAttribute] = [.top, .leading, .bottom, .trailing]
+    for view in nib.instantiate(withOwner: owner, options: nil) {
       if let view = view as? UIView {
         view.translatesAutoresizingMaskIntoConstraints = false
         owner.addSubview(view)
         layoutAttributes.forEach { attribute in
           owner.addConstraint(NSLayoutConstraint(item: view,
             attribute: attribute,
-            relatedBy: .Equal,
+            relatedBy: .equal,
             toItem: owner,
             attribute: attribute,
             multiplier: 1,
```

Comment:
multiplier is `CGFloat` so I would do `1.0` instead of `1`,  but it's not crucial ;)